### PR TITLE
CVE-2015-8213

### PR DESCRIPTION
**Pull request by ssneck2**
Completed write up of django CVE-2015-8213

### CODE REVIEW

File: cves/CVE-2015-8213.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: A getter function belonging to a utility file in Django existing before the patched version might allow remote attackers to obtain sensitive application secrets via a settings key in place of a date/time format setting. This allowed a malicious user to pass a secret key instead of a date format and obtain said application secret.
```
Comment (gollb310) [inline cves/CVE-2015-8213.yml]: Make sure to limit lines to 80 characters for readability.

File: cves/CVE-2015-8213.yml [hunk 2]
```diff
@@ -173,17 +173,17 @@ subsystem:
...
-  question: |
+  question: | 
     Are there any interesting commits between your VCC(s) and fix(es)?
 
     Write a brief (under 100 words) description of why you think this commit was
     interesting in light of the lessons learned from this vulnerability. Any
     emerging themes?
   commits:
-  - commit:
+  - commit:
```
Comment (gollb310) [inline cves/CVE-2015-8213.yml]: If you did not find any interesting commits, make sure to give a brief summary of other changes to the VVCs to show you did go through their commit history.

**Review by gollb310 — COMMENTED** [review]
Overall good, but you need some line formatting and section expansion. +1 upvote.

File: cves/CVE-2015-8213.yml (hunk 1 above)
Comment (cms2146) [reply cves/CVE-2015-8213.yml]: You switch tense in the middle of the desc here, I'd recommend keeping it all in present progressive

File: cves/CVE-2015-8213.yml [hunk 3]
```diff
@@ -173,17 +173,17 @@ subsystem:
     In the name field, a subsystem name (or an array of names)
 
     e.g. clipboard, model, view, controller, mod_dav, ui, authentication
-  answer:
-  name:
+  answer: Public post describing cve and fix on django development site.
+  name: utils
```
Comment (cms2146) [inline cves/CVE-2015-8213.yml]: General rule I've picked up from dev work, util folder/files are the work of the devil. Everything, by definition, has utility, so it isn't super descriptive. Maybe look back at the spec and try to find something better.

File: cves/CVE-2015-8213.yml [hunk 4]
```diff
@@ -105,10 +105,10 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: No units tests were used in the fix for this cve.
+  fix: false
```
Comment (ianrandman) [inline cves/CVE-2015-8213.yml:114]: It looks like a test was added in the fix commits. Also, you should put notes for the fix commits.

File: cves/CVE-2015-8213.yml [hunk 5]
```diff
@@ -268,7 +268,7 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: Requirement mistakes likely led to this vulnerability as trusted input is used
```
Comment (ianrandman) [inline cves/CVE-2015-8213.yml]: Was any other sanitation occurring, and this was just a spot they missed?

**Pull request merged**